core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -56,6 +63,8 @@
     </button>
   </div>
 </div>
+<div ng-show="modalWrapperVisible" ng-click=" modalWrapperVisible = false; " class="modal-wrapper">
```

Comment:
Place the variable assignments (`modalWrapperVisible`) behind a JS function, this is so that things are all in once place when we do introduce new variables and makes it easier to debug when things go wrong.